docs/source/conf.py

```diff
@@ -4,6 +4,26 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 import fastplotlib
 
+from sphinx_gallery.sorting import ExplicitOrder
+import wgpu.gui.offscreen
+import os
+import sys
+
+ROOT_DIR = os.path.abspath(os.path.join(__file__, "..", ".."))
+sys.path.insert(0, ROOT_DIR)
+
+# -- Sphix Gallery Hackz -----------------------------------------------------
+# When building the gallery, render offscreen and don't process
+# the event loop while parsing the example
+
+
+def _ignore_offscreen_run():
+    wgpu.gui.offscreen.run = lambda: None
```

Comment:
This bit should no longer be necessary. It's from a time when the offscreen backend still fired up asyncio (it does not do so anymore). The `WGPU_FORCE_OFFSCREEN` is the only thing needed.